src/Parser.php

```diff
@@ -40,7 +40,7 @@ private function parseParameters($text)
         $return = array();
         foreach($argsMatches[1] as $item)
             {
-            $parts = explode($this->syntax->getParameterValueSeparator(), $item, 2);
+            $parts = preg_split('~\s*' . $this->syntax->getParameterValueSeparator() . '\s*~', $item, 2);
```

Comment:
We'll go into PSR-2 after merging #5 and this. I just mentioned PSR-2 since you previously proposed PR for it, sorry for the ambiguity. I meant only to discuss spaces around `.` operator.